Framework/PythonInterface/plugins/algorithms/WorkflowAlgorithms/DirectILLAutoProcess.py

```diff
@@ -591,7 +591,7 @@ def _prepare_masks(self):
             existing_masks.append(tube_mask_ws)
 
         mask_angles = self.getProperty("MaskedAngles").value
-        if mask_angles != list():
```

Comment:
Is this intentional becuase I guess the behaviour is different?

Reply:
It was comparing `mask_angles` to an empty list, but `mask_angles` is a `numpy` array, and the new version didn't like being compared to a list. So instead we can call `.any()` on the array and the behaviour will be the same as before.